week-3/todo-list/index.html

```diff
@@ -3,25 +3,24 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Title here</title>
+    <title>To Do List</title>
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <form>
-      <div>
-        <input type="text" placeholder="New todo..." />
+    <nav>
+      <div class="container-title">TODO LIST</div>
+    </nav>
+    <div class="container">
+      <div class="todo-app">
+        <h2>Add new todo</h2>
+        <div class="row">
+          <input type="text" id="input-box" placeholder="New todo..." />
```

Comment:
This is a very good implementation. The only missing feature is that user's should be able to set a deadline for todo's when creating it. Overall great work!